### DISCUSSION TITLE
Yazi - Images

### DISCUSSION THREAD

**Discussion by peteringram0** [zellij-org/zellij#2831] [Q&A]
I have been using `yazi` recently which is a file manager. Yazi supports displaying images in the terminal which works fine locally for me however inside `Zellij` this does not seem to work. Im not sure how Yazi is doing this under the hood but assume they are doing something thats not supported inside Zellij?

https://yazi-rs.github.io/docs/showcase/

**Comment by imsnif** [comment]
Looks cool! Zellij supports Sixel, so this should work. I'd start by opening a bug in the Yazi code-base and seeing if they can provide more info.

**Reply by zetashift** [reply]
Should be fixed if somebody is on nightly: https://github.com/sxyazi/yazi/pull/181#issuecomment-1744307719
